tools/c7n_azure/c7n_azure/storage_utils.py

```diff
@@ -30,7 +29,7 @@
 class StorageUtilities(object):
```

Comment:
Should we just consider making `StorageUtilities` a non-static only method class and just instantiate the proper session it needs once? I feel like this will eliminate future bugs of people passing in the wrong session.

Reply:
It gets used outside the context of custodian a bunch so I avoid assuming we have the ability to make sessions correctly

Reply:
They can't pass in the wrong session any more though - it will accept *any* session and work fine now